tools/src/vendoring/config/expoGoConfig.ts

```diff
@@ -18,9 +18,6 @@ const config: VendoringTargetConfig = {
     },
   },
   modules: {
-    'amazon-cognito-identity-js': {
-      source: 'https://github.com/aws-amplify/amplify-js.git',
-    },
```

Comment:
we may keep this?

Reply:
I couldn't find any other reference to it 🤔

Reply:
having this we can keep amazon-cognito-identity-js appearing from `et uvm -l`. though its vendoring is tricky because its code lives in `aws-amplify`. it's not actively maintained so we don't update vendoring code often.